src/manageArtifacts.ts

```diff
@@ -36,11 +36,11 @@ const defineVariableOperation = (variable: string): VariableStatus => {
             throw Error(`Both key and value are empty`)
         }
     } catch (error) {
-        throw Error('Error type')
+        throw Error(`Type error: ${error}`);
```

Comment:
Why "Type error"? Why not just "Error"?

Reply:
An error here is not a random one, we know that it comes from the format of the inputs.

Reply:
Then something like "Input type error" would make more sense.